src/main/java/by/exadel/internship/team_four/entity/Form.java

```diff
@@ -0,0 +1,42 @@
+package by.exadel.internship.team_four.entity;
+
+
+import by.exadel.internship.team_four.entity.enums.EnglishLevel;
+import by.exadel.internship.team_four.entity.enums.FormStatus;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Form {
+
+    private UUID id;
+    private String name;
+    private String lastName;
+
+    //не обяз
+    private String middleName;
+    private String email;
+    private String phoneNumb;
+    private String skype;
+    private EnglishLevel englishLevel;
+    private String country;
+    private String city;
+
+    //не обяз
+    private String experience;
+
+    //private String primarySkill; спросить у Жанны
+
+    private String education;
+    private String filePath;
+    private Interview interview;
+    private FormStatus formStatus;
+    private TimeForCall timeForCall;
```

Comment:
Is it approved that UI send you only one time period? Or it's questionable?

Reply:
we would like to  ask Zanna about it ? but we hadnt meteeng today, and we are going to write her our questions. Now we decided to leave this one

Reply:
Please add comment that this field is questionable.

Reply:
done